doc/contexts-that-disable-exit-on-error.md

```diff
@@ -0,0 +1,87 @@
+Contexts That Disable Exit On Error
+===================================
+
+In wick, everything by default runs in [strict mode](bash-strict-mode.md).  This, among other things, sets `-e` which tells bash to exit if a command that was ran exitted with an error code.  It will also exit with the same error code as the command that failed.
```

Comment:
Here and in the other comment.   

Wick is capitalized, as is Bash.
Exited has one t.